aiohttp/client.py

```diff
@@ -478,12 +485,35 @@ def _ws_connect(self, url, *,
                         protocol = proto
                         break
 
+            # websocket compress
+            notakeover = False
+            if compress:
+                if hdrs.SEC_WEBSOCKET_EXTENSIONS in resp.headers:
```

Comment:
Use `resp.headers.get(hdrs.SEC_WEBSOCKET_EXTENSIONS)`, do getting the header only once.